.github/workflows/cypress.yml

```diff
@@ -93,8 +93,8 @@ jobs:
 
     - name: Log WP environment versions
       run: |
-        npx wp-env run cli "wp core version"
```

Comment:
We needed to remove quotes because, with quotes, arguments were not being passed properly. The container was looking for a "wp core version" binary.